lib/FluxibleContext.js

```diff
@@ -99,12 +99,27 @@ FluxContext.prototype.plug = function (plugin) {
 };
 
 /**
+ * Get action name, compatible with all browsers
+ * @param {Function} action
+ * @return {String} action display name
+ */
+function getActionName (action) {
+    var actionName = action.displayName || action.name;
+    if (!actionName) {
+        var match = action.toString().match(/^function\s*([^\s(]+)/);
```

Comment:
I think it's fine since actionName here is only debug usage, 
I generated this for some other purpose which is not valid anymore, just keep the code